tests/test_prompts.py

```diff
@@ -109,9 +109,7 @@ def test_llama3_instruct_prompt():
     config.few_shot_examples.num_few_shots = 2
     prompt = Prompt(config=config)
 
-    expected_prompt = """<|begin_of_text|><|start_header_id|>system<|end_header_id|>
-
-<|eot_id|><|start_header_id|>user<|end_header_id|>
+    expected_prompt = """<|start_header_id|>user<|end_header_id|>
```

Comment:
There are some tests that are failing because of the newly added bos tokens, can you please fix references there?